test/performance/odp_packet_gen.c

```diff
@@ -1115,12 +1229,69 @@ static inline int send_burst(odp_pktout_queue_t pktout, odp_packet_t pkt[],
 	if (odp_unlikely(sent != num)) {
 		uint32_t num_drop = num - sent;
 
+		for (i = sent; i < num; i++)
+			bytes_total -= odp_packet_len(pkt_ref[i]);
 		odp_packet_free_multi(&pkt_ref[sent], num_drop);
 	}
 
+	*sent_bytes = bytes_total;
 	return sent;
 }
 
+static int alloc_test_packets(odp_pool_t pool, odp_packet_t *pkt_tbl, int num_pkt,
+			      int pkts_per_pktio, test_options_t *test_options)
+{
+	int num_alloc = 0;
+
+	if (test_options->use_rand_pkt_len) {
+		int i, j;
+		int num_pktio = test_options->num_pktio;
+		uint32_t pkt_bins = test_options->rand_pkt_len_bins;
+		uint32_t pkt_len_min = test_options->rand_pkt_len_min;
+		uint32_t pkt_len_max = test_options->rand_pkt_len_max;
+		uint32_t bin_size = 1;
+
+		if (pkt_bins)
+			bin_size = (pkt_len_max - pkt_len_min + 1) / pkt_bins;
```

Comment:
Should division be done with (pkt_bins - 1)?

E.g. max = 20, min = 10, bins = 3: leeds now to packet lengths of 10, 13 and 20